docs/getting-started/flashing-firmware/nrf52/swdio.mdx

```diff
@@ -22,13 +22,15 @@ Connect GPIO pin 8 on your Raspbery Pi to the DIO pin on the Meshtastic Device.
 Ensure you get the GPIO numbers correct.
 
 Plug in your Raspberry Pi to USB power.
-Plug in the Meshtastic Device to power.
+Plug in the Meshtastic Device to power - to your Raspberry Pi's USB port.
 
 ## Install OpenOCD
 
 You can use OpenOCD to manage the SWDIO connection with the Meshtastic Device.
 Install it on your Raspberry Pi.
 
+You can also use the packet manager in Raspberry Pi OS (`sudo apt install openocd`).
```

Comment:
s/packet manager/package manager

Reply:
Will need to find where on the filesystem you can get the config files -- see line 50.